package/root/usr/local/sbin/resize_rootfs.sh

```diff
@@ -1,13 +1,24 @@
 #!/bin/bash
 
-set -xe
+if [ $# -eq 0 ]; then
```

Comment:
I was thinking that, but I didn't like do that without actually verifying that 0 is eMMC, 1 is SD, etc... don't like hard-coded paths without verification checks... smells of 'dumb luck' ;) Hence leave it to he user for simplicity for now ?